action.c

```diff
@@ -111,7 +114,14 @@
 #include "parserif.h"
 #include "statsobj.h"
 
+/* AIXPORT : cs renamed to cs as clashes with libpthreads variable in complete file*/
```

Comment:
as a side-note, ```cs_t``` is a bit misleading, because ```cs``` simply is a variable (but not a type). This would probably better be renamed to ```legacy_cs``` as it is only used for the legacy system.